exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/FunctionImplementationRegistry.java

```diff
@@ -86,6 +108,7 @@ public FunctionImplementationRegistry(DrillConfig config, ScanResult classpathSc
       }
     }
     logger.info("Function registry loaded.  {} functions loaded in {} ms.", drillFuncRegistry.size(), w.elapsed(TimeUnit.MILLISECONDS));
+    this.remoteFunctionRegistry = new RemoteFunctionRegistry(new UnregistrationListener());
```

Comment:
I figured out that not, since it's just class initialization and it doesn't load any functions.